tservice/src/seal.rs

```diff
@@ -3,18 +3,32 @@
 
 use alloc::{vec, vec::Vec};
 pub use core::ptr;
+use mc_sgx_core_types::{Attributes, KeyPolicy, MiscellaneousSelect};
 pub use mc_sgx_core_types::{Error, Result};
 pub use mc_sgx_tservice_sys::sgx_calc_sealed_data_size;
 pub use mc_sgx_tservice_sys_types::sgx_sealed_data_t;
 pub use mc_sgx_tservice_types::Sealed;
 use mc_sgx_util::ResultInto;
 
+// Default values used in `sgx_seal_data_ex` to behave the same as
+// `sgx_seal_data`. See
+// https://download.01.org/intel-sgx/sgx-linux/2.17.1/docs/Intel_SGX_Developer_Reference_Linux_2.17.1_Open_Source.pdf
+const DEFAULT_MISCELLANEOUS_MASK_FOR_SEAL: u32 = 0xF0000000;
+const DEFAULT_ATTRIBUTES_FLAGS_FOR_SEAL: u64 = 0xFF0000000000000B;
```

Comment:
This is `TSEAL_DEFAULT_FLAGSMASK` in `common/inc/internal/tseal_migration_attr.h`, if you'd prefer to use that.

Reply:
Do you mean directly from the header, or using the name for similarity between the code bases?
- The header is _internal_ implied to be private, so don't think we should use it.
- These are local consts so could name them similarly.  I think for now I'll add a comment referencing the internal name.  If one does feel the names are too long I could understand that.  I usually fall into the verbose over abbreviated camp, but know that it can be taken too far at times.